# Service to get forward kinematics of the robot
float64[] joints
---
float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz
bool success
string msg
